.github/workflows/slash-commands.yml

```
@@ -0,0 +1,34 @@
+          DAGGER_CLOUD_TOKEN: ${{ secrets.DAGGER_CLOUD_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+      - name: slash review
+        if: github.event.issue.pull_request && contains(github.event.comment.body, '/review')
+        run: dagger call pull-request-review --github-token GH_TOKEN --issue-id ${{ github.event.issue.number }}
+        env:
+          DAGGER_CLOUD_TOKEN: ${{ secrets.DAGGER_CLOUD_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+      - name: slash develop
+        if: ${{ !github.event.issue.pull_request }} && contains(github.event.comment.body, '/develop')
```

actionlint (1.7.7)

if: condition "${{ !github.event.issue.pull_request }} && contains(github.event.comment.body, '/develop')" is always evaluated to true because extra characters are around ${{ }}

(if-cond)